Xamarin.Forms.Core/ExportFontAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+namespace Xamarin.Forms
+{
+	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
+	public class ExportFontAttribute : Attribute
```

Comment:
is an attribute the best way to register a new font ? I'm thinking about App level resources...

Reply:
I am not married to the attribute. I personally don't even like them for renderers, but this allows them to be shipped in nugets/dlls without the user needing to do anything.